.travis.yml

```
@@ -1,24 +1,33 @@
 env:
   - USER=travis GO111MODULE=off
 
-os:
-  - osx
-
 language: go
+go:
+  - "1.13.x"
 
 branches:
   only:
     - master
 
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
+
 jobs:
   fast_finish: true
   include:
-    - go: "1.13.x"
-      name: "go test"
+    - os: osx
+      name: "go test coverage osx"
       script:
         - df -h
         - travis_wait make ci
-    - go: "1.13.x"
-      name: "go lint"
-      script: travis_wait make ci-lint
-
+        - travis_wait 20 bash scripts/travis.sh
```

Could `travis_wait make ci` simply be changed to :
```suggestion
        - travis_wait TESTARGS="-coverprofile=coverage.txt  -covermode=atomic" make ci
```

So that we don't need any `scripts/travis.sh` ? Ah, may be we need to loop on every package ?

---

I'm using the `travis.sh` because of this https://github.com/codecov/example-go#caveat-multiple-files.
I can try this other way in travis anyways!

---

Got it ! :)